wordpress-mobile/react-native-libraries-publisher
# Bump `react-native`  to version `0.71.8`

Bumps the React Native version to `0.71.8`. This also implied updating the module substitution block to reflect the new [process for fetching the React Native and Hermes modules](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0508-out-of-npm-artifacts.md#implemented-solution).

## How to test
1. Run the command `npm install`.
2. Run the command `./gradlew publishToMavenLocal -exclude-task prepareToPublishToS3`.
3. Observe that all libraries are built and published successfully in the local folder `~/.m2/repository/org/wordpress-mobile/react-native-libraries`.
